PYME/recipes/processing.py

```diff
@@ -2905,4 +2905,59 @@ def applyFilter(self, data, chanNum, i, image0):
         if self.offset_selection == 'offset by a constant':
             return data - self.offset_constant
         elif self.offset_selection == 'offset by minimum':
-            return data - np.min(data)
\ No newline at end of file
+            return data - np.min(data)
+
+
+@register_module('SubPixelDriftCorrect')
+class SubPixelDriftCorrect(ModuleBase):
+    """
+    Sub-pixel lateral drift correction based on the drift tracking events
+    in the transmitted-light channel
+
+    Parameters
+    ----------
+    input : Input
+        PYME.IO.ImageStack
+    drift_tracking_channel_pixel_size_x : Float
+        Pixel value of the drift tracking channel in x, in nm unit
+    drift_tracking_channel_pixel_size_y : Float
+        Pixel value of the drift tracking channel in y, in nm unit
+    relative_rotation : Float
+        relative rotation degree from the drift tracking channel to the channel to be corrected, in degree
+
+    Returns
+    -------
+    output = Output
+        PYME.IO.ImageStack
+
+    """
+
+    input = Input('input')
+    drift_tracking_channel_pixel_size_x = Float(64.5)
```

Comment:
In many cases the drift-tracking camera will be flipped as well as rotated with respect to the imaging camera. The general case (as is used for point-data) is to specify an affine transformation matrix, rather than pixel sizes and rotations, although that is arguably a bit less user-friendly. 

I'd need to double check, but it's likely that the necessary transformation can be spoofed with the pixel-size and angle if we allow negative pixel sizes. If this is the case we need to think about whether we want to be consistent with the point data, or whether the pixel size and angle specification is sufficiently more user friendly to make it a better choice.